app/presenters/auction_presenter.rb

```diff
@@ -151,6 +151,15 @@ def auction_rules
     @auction_rules ||= RulesFactory.new(self).create
   end
 
+  def small_business?
+    start_price > Auction::MICROPURCHASE_THRESHOLD
+  end
+
+  def eight_a_stars
+    # this could be a db column
+    true
+  end
+
```

Comment:
Following the discussion above, we would probably eliminate both of these boolean methods and instead have a factory that returns the right `Eligibility` class based on one or more conditions in the auction.